controllers/reviews.controllers.js

```diff
@@ -7,4 +7,15 @@ exports.getSingleReview = (req, res, next) => {
         res.status(200).send({ review });
     })
     .catch(next);
+};
+
+exports.patchReview = (req, res, next) => {
+    const review_id = req.params;
+    selectReview(review_id)
+    .then((review) => {
+        console.log(req.body);
+        review.votes += req.body.inc_votes;
```

Comment:
You need to update the votes in the database. At the moment this just changes a value of the object, but the votes remain the same in the db. 

look at creating a model and how to update values in sql.